Update nf-fileapi-getfinalpathnamebyhandlea.md

cstraessner opened the pull request:
adding note  and reference  to SMB2 specs  in regard  of  support for FileNormalizedNameInformation

cstraessner committed:
Update nf-fileapi-getfinalpathnamebyhandlea.md

adding note  and reference  to SMB2 specs  in regard  of  support for FileNormalizedNameInformation

alvinashcraft commented on sdk-api-src/content/fileapi/nf-fileapi-getfinalpathnamebyhandlea.md:
```diff
@@ -233,6 +233,11 @@ Invalid flags were specified for <i>dwFlags</i>.
 
 The Server Message Block (SMB) Protocol does not support queries for normalized paths. Consequently, when you call this function passing the handle of a file opened using SMB, and with the FILE_NAME_NORMALIZED flag, the function splits the path into its components and tries to query for the normalized name of each of those components in turn. If the user lacks access permission to any one of thos ...
 
+Windows Versions  post Windows 10 v1709 and Windows Server v1709 do support FileNormalizedNameInformation  information class  via  SMB per  the MS-SMB2 specification Appendix A , Section 3.3.5.20.1
```
Updating to use standard verbiage for referencing Windows versions and updating link to remove base URL and language/region. Also calling this out with the NOTE alert format.

```suggestion
> [!NOTE]
> Windows 10 version 1709 and later and Windows Server version 1709 and later support the **FileNormalizedNameInformation** information class via SMB. See the [MS-SMB2 specification] (/openspecs/windows_protocols/ms-smb2/a64e55aa-1152-48e4-8206-edd96444e7f7#Appendix_A_414) Appendix A, Section 3.3.5.20.1 for more information.
```

alvinashcraft commented on sdk-api-src/content/fileapi/nf-fileapi-getfinalpathnamebyhandlea.md:
```diff
@@ -233,6 +233,11 @@ Invalid flags were specified for <i>dwFlags</i>.
 The Server Message Block (SMB) Protocol does not support queries for normalized paths. Consequently, when you call this function passing the handle of a file opened using SMB, and with the FILE_NAME_NORMALIZED flag, the function splits the path into its components and tries to query for the normalized name of each of those components in turn. If the user lacks access permission to any one of thos ...
 
+Windows Versions  post Windows 10 v1709 and Windows Server v1709 do support FileNormalizedNameInformation  information class  via  SMB per  the MS-SMB2 specification Appendix A , Section 3.3.5.20.1
+<a href="https://learn.microsoft.com/en-us/openspecs/windows_protocols/ms-smb2/a64e55aa-1152-48e4-8206-edd96444e7f7#Appendix_A_414"> MS-SMB2 specification</a>
```
```suggestion
```

alvinashcraft commented on sdk-api-src/content/fileapi/nf-fileapi-getfinalpathnamebyhandlea.md:
```diff
@@ -233,6 +233,11 @@ Invalid flags were specified for <i>dwFlags</i>.
 
+Windows Versions  post Windows 10 v1709 and Windows Server v1709 do support FileNormalizedNameInformation  information class  via  SMB per  the MS-SMB2 specification Appendix A , Section 3.3.5.20.1
+<a href="https://learn.microsoft.com/en-us/openspecs/windows_protocols/ms-smb2/a64e55aa-1152-48e4-8206-edd96444e7f7#Appendix_A_414"> MS-SMB2 specification</a>
+
```
```suggestion
```

alvinashcraft commented on sdk-api-src/content/fileapi/nf-fileapi-getfinalpathnamebyhandlea.md:
```diff
@@ -233,6 +233,11 @@ Invalid flags were specified for <i>dwFlags</i>.
+Windows Versions  post Windows 10 v1709 and Windows Server v1709 do support FileNormalizedNameInformation  information class  via  SMB per  the MS-SMB2 specification Appendix A , Section 3.3.5.20.1
+<a href="https://learn.microsoft.com/en-us/openspecs/windows_protocols/ms-smb2/a64e55aa-1152-48e4-8206-edd96444e7f7#Appendix_A_414"> MS-SMB2 specification</a>
+
+
```
```suggestion
```
Removing extra line breaks added.

alvinashcraft reviewed (changes requested):
Added some suggestions inline. Thanks for the PR!

alvinashcraft committed:
Apply suggestions from code review

I'm applying the suggestions. I'll wait to merge until tomorrow in case there are other suggestions.

alvinashcraft commented:
This same note should probably be added to [GetfFinalPathNameByHandleW](https://learn.microsoft.com/en-us/windows/win32/api/fileapi/nf-fileapi-getfinalpathnamebyhandlew).
